src/lib/tradingView/utils.ts

```diff
@@ -224,10 +226,10 @@ export const getWidgetOptions = () => {
       'header_symbol_search',
       'header_compare',
       'symbol_search_hot_key',
-      'compare_symbol',
       'symbol_info',
       'go_to_date',
       'timeframes_toolbar',
+      'header_layouttoggle',
```

Comment:
this is the multi-chart feature, which we can enable when we want to implement the feature fully